istio/restrict-virtual-service-wildcard/restrict-virtual-service-wildcard.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-virtual-service-wildcard
+  annotations:
+    policies.kyverno.io/title: Restrict Virtual Service Host with Wildcards
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    kyverno.io/kyverno-version: 1.6.2
```

Comment:
This annotation holds the version of Kyverno on which it was tested and confirmed to work. Did you not use one of the 1.8 releases to test this?

Reply:
sorry about that, i tested with version 1.8.4 on Openshift 4.10. Fixed in the current push